python/ray/serve/_private/replica.py

```diff
@@ -673,11 +675,12 @@ async def reconfigure(
 
     def _get_metadata(
         self,
-    ) -> Tuple[DeploymentConfig, DeploymentVersion, Optional[float]]:
+    ) -> Tuple[DeploymentConfig, DeploymentVersion, Optional[float], Optional[int]]:
```

Comment:
Can we update the type hint for `initialize_and_get_metadata()` and `reconfigure()` as well? 😅

Reply:
thanks for the catch, updated